### DISCUSSION TITLE
Apply snapshot in addition

### DISCUSSION THREAD

**Discussion by Jbmanllr** [directus/directus#13599] [Feature Requests]
Would it be possible to allow us to apply a snapshot in addition of the current data model, instead of replacing everything? 
For example it could have a "blog" snapshot that i would just add over my existing db design.

**Comment by azrikahar** [comment]
It'd be a great addition indeed (pun intended). There are similar discussions as well such as #11262, #11249 and #9214.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
